### PR TITLE
Add image and random fun fact to index.html 

### PR DESCRIPTION
**Pull request by AshleyLTW**
Insert image tags into index.html to show a large photo of my dog, also modified the script.js file to display random fun facts about me instead of multilingual greetings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,3 @@
...
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This repo contains Ashley's portfolio and STEP projects
```
Comment (darrnshn) [inline README.md]: nit: full stop at the end?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,22 @@
...
       <p>This is my portfolio.</p>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <p>Hi! My name is Ashley, I'm a rising Junior studying CS at Harvard. 
+        I enjoy dancing, watching my brother bake (and claiming the results for myself) as well as taking long walks with my dog.</p>
     </div>
+
+    <img src="./images/dog-bundle.jpg" alt="A toy poodle being carried in her dog bed">
+
```
Comment (darrnshn) [inline portfolio/src/main/webapp/index.html:20]: nit: no need for blank line here

**Review by darrnshn — CHANGES_REQUESTED** [review]
Looks good! A couple of nitpicks.

**Comment by AshleyLTW** [comment]
I've committed the changes!

**Pull request merged**